src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java

```diff
@@ -35,11 +39,18 @@ public class StatTypeValue implements Value<Number>, Parser {
         }
     }
 
-    private Stats.General getKeyFromString(String key) {
-        for (Stats.General stat : Stats.General.values()) {
-            if (stat.name().equalsIgnoreCase(key)) return stat;
-        }
-        return null;
+    private StatsAPI.Key getKeyFromString(String token) {
+        String[] tokenParts = token.split(":");
+
+        String statNamespace = tokenParts.length == 3 ? tokenParts[0] : null;
```

Comment:
if this turns out to be an empty string, it needs to be treated as null, eg: `:general:uridium` would reference namespace = null, category = general, key = uridium

Reply:
As it stands right now it is backwards compatible and as far as I can see it works correctly.

Reply:
In case of `:general:uridium` it will try to match against namespace=`""`, category=`"general"` and stat=`"uridium"`, which will find nothing in that "" namespace.
Just make it so if it's an empty string it's also treated as null